stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -628,4 +638,97 @@ protected void closeInStream() {
       }
     }
   }
+
+  private final class AlluxioPOSIXBenchThread extends BenchThread {
+
+    private RandomAccessFile mRandomAccessFile = null;
+
+    private AlluxioPOSIXBenchThread(BenchContext context, int threadId) {
+      super(context, threadId);
+    }
+
+    @Override
+    @SuppressFBWarnings("BC_UNCONFIRMED_CAST")
+    protected int applyOperation() throws IOException, AlluxioException {
+      if (ClientIOOperation.isRead(mParameters.mOperation)) {
+        if (mRandomAccessFile == null) {
+          mRandomAccessFile = new RandomAccessFile(mFilePath.toString(), "r");
```

Comment:
since it's RandomAccessFile, we can open it in "rw" for all read and write related operations?

Reply:
Yes. We also don't have to check whether it's null inside different operations